client/__tests__/util/dataframe/dataframe.test.js

```diff
@@ -0,0 +1,296 @@
+import * as Dataframe from "../../../src/util/dataframe";
+
+/*
+Known gaps in these tests:
+- row indexing, especially a la 'world', are not sufficiently tested
```

Comment:
I would like the `at/iat` methods to be at least cursorily tested too.

Reply:
will add